HousingManagementSystemApi.Tests/UseCasesTests/RetrieveAddressesUseCaseTests.cs

```diff
@@ -44,13 +56,16 @@ public async Task GivenAPostcode_WhenExecute_GatewayReceivesCorrectInput()
         public async Task GivenAPostcode_WhenAnAddressExistsWithAnEligibleAssetType_ThenThePropertyIsReturned()
         {
             const string TestPostcode = "postcode";
-            retrieveAddressesGateway.Setup(x => x.SearchByPostcode(TestPostcode))
+            this.retrieveAddressesGateway.Setup(x => x.SearchByPostcode(TestPostcode))
                 .ReturnsAsync(new PropertyAddress[] { new() { PostalCode = TestPostcode, Reference = new Reference { ID = "assetId" } } });
 
-            retrieveAssetGateway.Setup(x => x.RetrieveAsset("assetId"))
-                .ReturnsAsync(new AssetResponseObject { AssetType = AssetType.Dwelling });
+            this.retrieveAssetGateway.Setup(x => x.RetrieveAsset("assetId"))
```

Comment:
Nitpick: Please remove `this` qualifier
```suggestion
            retrieveAssetGateway.Setup(x => x.RetrieveAsset("assetId"))
```